src/screens/EmbeddedViewer.js

```diff
@@ -24,16 +26,18 @@ export default function ScreensEmbeddedViewer() {
   return (
     <div className="embeded-viewer-wrapper" css={embedWrapper}>
       <img src={logo} alt="Northwestern logo" css={imgLogo} />
-      <OpenSeadragonViewer
-        manifestUrl={decodeURIComponent(params.manifestUrl)}
-        options={{
-          showDropdown: true,
-          showThumbnails: true,
-          showToolbar: true,
-          deepLinking: false,
-          height: 800,
-        }}
-      />
+      <ErrorBoundary FallbackComponent={FallbackErrorComponent}>
```

Comment:
Can we make FallbackErrorComponent the default instead of fetching the component in every file?

Reply:
@divyareddynu I'd like to keep it like this, because there are some situations where we'll want to pass in a custom fallback error component instead of this default one.  Like say we want it specifically to say "Something went wrong in the Work Preservation Tab", etc.   Because this issue only applies the ErrorBoundary component to "screen" components, I just blanket applied it.  But it will get more granular in the future I'm guessing.